dsl_parser/tests/test_deafult_import_resolver.py

```diff
@@ -20,7 +20,7 @@
 
 from dsl_parser.exceptions import DSLParsingLogicException
 from dsl_parser.import_resolver.default_import_resolver import \
-    DefaultImportResolver
+    DefaultImportResolver, DefaultResolverValidationException
 
 ORIGINAL_V1_URL = 'http://www.original_v1.org/cloudify/types.yaml'
```

Comment:
what's the meaning of this url? (looks like it can actually be accessible, it's redirected to "http://www.original_v1.org/?pid=9PO312GO9&dn=originalv1.org&rpid=5PO84Q269")